-------------------------------------------------------------------------------
Test set: dk.sdu.mmmi.cbse.asteroidsnetbeansmodules.ApplicationTest
-------------------------------------------------------------------------------
Tests run: 2, Failures: 1, Errors: 1, Skipped: 0, Time elapsed: 11.186 sec <<< FAILURE!
testApplication(dk.sdu.mmmi.cbse.asteroidsnetbeansmodules.ApplicationTest)  Time elapsed: 10.037 sec  <<< ERROR!
java.nio.file.NoSuchFileException: \Users\corfixen\Workspaces\GitHub\SB4-KOM\AsteroidsNetbeansModules\AsteroidsNetbeansModules\application\src\test\resources\enemy\updates.xml
	at sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:79)
	at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:97)
	at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:102)
	at sun.nio.fs.WindowsFileCopy.copy(WindowsFileCopy.java:99)
	at sun.nio.fs.WindowsFileSystemProvider.copy(WindowsFileSystemProvider.java:278)
	at java.nio.file.Files.copy(Files.java:1274)
	at dk.sdu.mmmi.cbse.asteroidsnetbeansmodules.ApplicationTest.testApplication(ApplicationTest.java:52)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at junit.framework.TestCase.runTest(TestCase.java:176)
	at org.netbeans.junit.NbTestCase.access$200(NbTestCase.java:102)
	at org.netbeans.junit.NbTestCase$2.doSomething(NbTestCase.java:478)
	at org.netbeans.junit.NbTestCase$1Guard.run(NbTestCase.java:404)
	at org.netbeans.junit.NbTestCase.runBare(NbTestCase.java:497)
	at junit.framework.TestResult$1.protect(TestResult.java:122)
	at junit.framework.TestResult.runProtected(TestResult.java:142)
	at junit.framework.TestResult.run(TestResult.java:125)
	at junit.framework.TestCase.run(TestCase.java:129)
	at org.netbeans.junit.NbTestCase.run(NbTestCase.java:293)
	at junit.framework.TestSuite.runTest(TestSuite.java:252)
	at org.netbeans.junit.NbModuleSuite$NbTestSuiteLogCheck.runTest(NbModuleSuite.java:1618)
	at junit.framework.TestSuite.run(TestSuite.java:247)
	at junit.framework.TestSuite.runTest(TestSuite.java:252)
	at junit.framework.TestSuite.run(TestSuite.java:247)
	at org.netbeans.junit.NbModuleSuite$S.runInRuntimeContainer(NbModuleSuite.java:949)
	at org.netbeans.junit.NbModuleSuite$S.access$100(NbModuleSuite.java:670)
	at org.netbeans.junit.NbModuleSuite$S$1.protect(NbModuleSuite.java:691)
	at junit.framework.TestResult.runProtected(TestResult.java:142)
	at org.netbeans.junit.NbModuleSuite$S.run(NbModuleSuite.java:687)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
testNoWarningsReportedDuringExecution(org.netbeans.junit.internal.NbModuleLogHandler$FailOnException)  Time elapsed: 10.059 sec  <<< FAILURE!
junit.framework.AssertionFailedError: NbModuleSuite has been started with failOnMessage(WARNING) and failOnException(INFO). The following failures have been captured:
[org.netbeans.modules.autoupdate.silentupdate] THREAD: pool-1-thread-1 MSG: A problem caught while refreshing Update Centers, cause: 
java.io.IOException: \Users\corfixen\netbeans_site\updates.xml (Den angivne sti blev ikke fundet)
  org.netbeans.modules.autoupdate.updateprovider.DownloadListener.notifyException(DownloadListener.java:103)
  org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogCache.copy(AutoupdateCatalogCache.java:246)
  org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogCache.writeCatalogToCache(AutoupdateCatalogCache.java:99)
  org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogProvider.refresh(AutoupdateCatalogProvider.java:154)
  org.netbeans.modules.autoupdate.services.UpdateUnitProviderImpl.refresh(UpdateUnitProviderImpl.java:180)
  org.netbeans.api.autoupdate.UpdateUnitProvider.refresh(UpdateUnitProvider.java:196)
  org.netbeans.modules.autoupdate.silentupdate.UpdateHandler.refreshSilentUpdateProvider(UpdateHandler.java:236)
  org.netbeans.modules.autoupdate.silentupdate.UpdateHandler.checkAndHandleUpdates(UpdateHandler.java:54)
  org.netbeans.modules.autoupdate.silentupdate.UpdateActivator$1.run(UpdateActivator.java:28)
  java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
  java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
  java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
  java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
  java.lang.Thread.run(Thread.java:748)
Caused by: java.io.FileNotFoundException: \Users\corfixen\netbeans_site\updates.xml (Den angivne sti blev ikke fundet)
  java.io.FileInputStream.open0(Native Method)
  java.io.FileInputStream.open(FileInputStream.java:195)
  java.io.FileInputStream.<init>(FileInputStream.java:138)
  java.io.FileInputStream.<init>(FileInputStream.java:93)
  sun.net.www.protocol.file.FileURLConnection.connect(FileURLConnection.java:90)
  sun.net.www.protocol.file.FileURLConnection.getInputStream(FileURLConnection.java:188)
  org.netbeans.modules.autoupdate.updateprovider.NetworkAccess$Task$2.call(NetworkAccess.java:180)
  org.netbeans.modules.autoupdate.updateprovider.NetworkAccess$Task$2.call(NetworkAccess.java:150)
  java.util.concurrent.FutureTask.run(FutureTask.java:266)
  ... 3 more

[org.netbeans.modules.autoupdate.silentupdate] THREAD: pool-1-thread-1 MSG: A problem caught while refreshing Update Centers, cause: 
java.io.IOException: \Users\corfixen\netbeans_site\updates.xml (Den angivne sti blev ikke fundet)
  org.netbeans.modules.autoupdate.updateprovider.DownloadListener.notifyException(DownloadListener.java:103)
  org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogCache.copy(AutoupdateCatalogCache.java:246)
  org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogCache.writeCatalogToCache(AutoupdateCatalogCache.java:99)
  org.netbeans.modules.autoupdate.updateprovider.AutoupdateCatalogProvider.refresh(AutoupdateCatalogProvider.java:154)
  org.netbeans.modules.autoupdate.services.UpdateUnitProviderImpl.refresh(UpdateUnitProviderImpl.java:180)
  org.netbeans.api.autoupdate.UpdateUnitProvider.refresh(UpdateUnitProvider.java:196)
  org.netbeans.modules.autoupdate.silentupdate.UpdateHandler.refreshSilentUpdateProvider(UpdateHandler.java:236)
  org.netbeans.modules.autoupdate.silentupdate.UpdateHandler.checkAndHandleUpdates(UpdateHandler.java:54)
  org.netbeans.modules.autoupdate.silentupdate.UpdateActivator$1.run(UpdateActivator.java:28)
  java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
  java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
  java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
  java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
  java.lang.Thread.run(Thread.java:748)
Caused by: java.io.FileNotFoundException: \Users\corfixen\netbeans_site\updates.xml (Den angivne sti blev ikke fundet)
  java.io.FileInputStream.open0(Native Method)
  java.io.FileInputStream.open(FileInputStream.java:195)
  java.io.FileInputStream.<init>(FileInputStream.java:138)
  java.io.FileInputStream.<init>(FileInputStream.java:93)
  sun.net.www.protocol.file.FileURLConnection.connect(FileURLConnection.java:90)
  sun.net.www.protocol.file.FileURLConnection.getInputStream(FileURLConnection.java:188)
  org.netbeans.modules.autoupdate.updateprovider.NetworkAccess$Task$2.call(NetworkAccess.java:180)
  org.netbeans.modules.autoupdate.updateprovider.NetworkAccess$Task$2.call(NetworkAccess.java:150)
  java.util.concurrent.FutureTask.run(FutureTask.java:266)
  ... 3 more


	at org.netbeans.junit.internal.NbModuleLogHandler.checkFailures(NbModuleLogHandler.java:218)
	at org.netbeans.junit.internal.NbModuleLogHandler$FailOnException.run(NbModuleLogHandler.java:240)
	at junit.framework.TestSuite.runTest(TestSuite.java:252)
	at junit.framework.TestSuite.run(TestSuite.java:247)
	at org.netbeans.junit.NbModuleSuite$S.runInRuntimeContainer(NbModuleSuite.java:949)
	at org.netbeans.junit.NbModuleSuite$S.access$100(NbModuleSuite.java:670)
	at org.netbeans.junit.NbModuleSuite$S$1.protect(NbModuleSuite.java:691)
	at junit.framework.TestResult.runProtected(TestResult.java:142)
	at org.netbeans.junit.NbModuleSuite$S.run(NbModuleSuite.java:687)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:113)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
